src/FlagBag.php

```diff
@@ -57,6 +51,24 @@ public function __construct(string $enumType, int ...$bits)
         $this->bits = static::decodeBits($bits);
     }
 
+    /**
+     * @param class-string $enumType
+     *
+     * @throws InvalidArgumentException
+     *
+     * @return void
+     */
+    public static function checkIntBackedEnumType(string $enumType)
```

Comment:
Let's keep this private for now

```suggestion
    private static function checkIntBackedEnumType(string $enumType): void
```